script/lib/importer.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+
+BINARIES = {
+  'darwin': [
+    'libbookmarks_browser.a',
+    'libcomponent_metrics_proto.a',
+    'libfavicon_base.a',
+    'libgoogle_core_browser.a',
+    'libhistory_core_browser.a',
+    'libsearch_engines.a',
+    'libquery_parser.a',
+    'liburl_formatter.a',
+  ],
+  'linux': [
```

Comment:
@bridiver I know he has tested w/ Windows for sure and I believe Linux was tested w/ Ubuntu.  Is that right, @darkdh?